app/assets/javascripts/views/history_view.js

```diff
@@ -5,41 +5,75 @@ module.exports = Backbone.View.extend({
   initialize: function() {
     _.bindAll(this, 'addActivities', 'connectionError');
 
-    this.currentStory = {};
-
-    this.$el.html(this.template({id: 'history', name: I18n.t('projects.show.history')}));
+    this.$el.html(this.template({
+      id: 'history',
+      className:'historycolumn',
```

Comment:
Agreed. :confused: what do you guys think of changing `.historycolumn` to `.activitycolumn`? It could match better with its content... but I think it could be more confusing than `.historycolumn`